### PR TITLE
Comment likes

### PR DESCRIPTION
**Pull request by ismaeelBM**

### CODE REVIEW

File: unhindled/connect.py [hunk 1]
```diff
@@ -278,27 +304,67 @@ def get_likes_on_post(post):
             auth = server[1]
             endpoint = "https://" + server[0] + "/author/" + author_id + "/posts/" + post_id + "/likes"
             req = requests.get(endpoint, auth=auth, headers={'Referer': "http://127.0.0.1:8000/"})
+            print(req.json())
```
Comment (PhamilyExpress) [inline unhindled/connect.py:307]:
```suggestion
 
```

File: unhindled/templates/unhindled/view_post.html [hunk 2]
```diff
@@ -41,9 +41,9 @@
 
 {% if user.is_authenticated %}
   {% if post|post_liked:user %}
-    <span class="mr-2"><a href="/author/{% valid_url_profile post.author.id %}/unlike/{% valid_url_profile post.id %}/post">{% post_text post user %}</a></span>
```
Comment (PhamilyExpress) [inline unhindled/templates/unhindled/view_post.html:44]: we changed the url?

**Review by PhamilyExpress — COMMENTED** [review]
looks good other than the comments

File: unhindled/templates/unhindled/view_post.html (hunk 2 above)
Comment (ismaeelBM) [reply unhindled/templates/unhindled/view_post.html:44]: yah previously the same link was for post and comment so I seperated them

**Pull request merged**